### ISSUE TITLE
Color picker not accessible

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#31637]
Refs: #31457

It is good that the color picker can be triggered via our "Show Hover" command, however after that you can not do any changes to it using the keyboard.
It should be possible to use the keyboard to navigate inside the color picker and choose colors appropriatly.

**Comment by cleidigh** [comment]
@isidorn 
@joaomoreno 
@rebornix 

I would love to use the new color picker but I can't without keyboard support :-(
Would you accept a PR for this keyboard controller?  

If so I'll make a proposal for the commands and approach...

**Comment by joaomoreno** [comment]
@cleidigh Unfortunately you'll have to wait for @rebornix to return, he's currently on leave. But feel free to start crafting a PR which makes your life easier yeah! 👍

**Comment by rbenson** [comment]
Any updates to this?

**Comment by cleidigh** [comment]
@rbenson 
No update yet.  Since accessibility is getting more more attention , I would assume this can get in the schedule sometime soon.

**Comment by isidorn** [comment]
This should work when the Color Picker is triggered via Show Hover command the focus should be passed into the color hover. Once focus is there:
* left, right, up, down should move the color 
* tab should allow to move focus to transparency

@rebornix might this be a good feature request for help-wanted tag? I am adding it and you can remove it.

**Comment by ShahzaibP** [comment]
Since the help-wanted tag is on now, can I possibly take a look into this issue @isidorn?

**Comment by isidorn** [comment]
@ShahzaibP I think yes. But it would be cool if @rebornix also gives his approval :)

**Comment by ShahzaibP** [comment]
Sure thing, I will wait for the approval!

**Comment by rebornix** [comment]
You are being kind, you don't need my approval for a contribution, go ahead ❤️

**Comment by lynxbutton** [comment]
I would like to work on this issue, is it still available to look into?
